test/unit/spec/MfaVerify_spec.js

```diff
@@ -1467,6 +1476,253 @@ function (Okta,
+          });
+        });
+        itp('has a passCode field', function () {
+          return setupEmail().then(function (test) {
+            expectHasAnswerField(test);
+          });
+        });
+        itp('has remember device checkbox', function () {
+          return setupEmail().then(function (test) {
+            Expect.isVisible(test.form.rememberDeviceCheckbox());
+          });
+        });
+        itp('clears the passcode text field on clicking the "Send email" button', function () {
+          return setupEmail().then(function (test) {
+            test.button = test.form.emailSendCode();
+            test.form.setAnswer('123456');
+            test.setNextResponse(resChallengeEmail);
+            expect(test.button.trimmedText()).toEqual('Send email');
+            expect(test.form.answerField().val()).toEqual('123456');
+            test.form.emailSendCode().click();
+            return tick().then(function () {
+              expect(test.button.trimmedText()).toEqual('Sent');
+              expect(test.form.answerField().val()).toEqual('');
+              var button = test.form.submitButton();
+              var buttonClass = button.attr('class');
+              expect(buttonClass).not.toContain('link-button-disabled');
+              expect(button.prop('disabled')).toBe(false);
+              return test;
+            });
+          });
+        });
+        itp('calls verifyFactor with empty code if send code button is clicked', function () {
+          return setupEmail().then(function (test) {
+            $.ajax.calls.reset();
+            test.setNextResponse(resChallengeEmail);
+            test.form.emailSendCode().click();
+            return tick();
+          })
+          .then(function () {
+            expect($.ajax.calls.count()).toBe(1);
+            Expect.isJsonPost($.ajax.calls.argsFor(0), {
+              url: 'https://foo.com/api/v1/authn/factors/emailhp9NXcoXu8z2wN0g3/verify',
+              data: {
+                passCode: '',
+                stateToken: 'testStateToken'
+              }
+            });
+          });
+        });
+        itp('calls verifyFactor with rememberDevice URL param', function () {
+          return setupEmail().then(function (test) {
+            $.ajax.calls.reset();
+            test.form.setRememberDevice(true);
+            test.setNextResponse(resChallengeEmail);
+            test.form.emailSendCode().click();
+            return tick();
+          })
+          .then(function () {
+            expect($.ajax.calls.count()).toBe(1);
+            Expect.isJsonPost($.ajax.calls.argsFor(0), {
+              url: 'https://foo.com/api/v1/authn/factors/emailhp9NXcoXu8z2wN0g3/verify?rememberDevice=true',
+              data: {
+                passCode: '',
+                stateToken: 'testStateToken'
+              }
+            });
+          });
+        });
+        itp('calls verifyFactor with empty code if verify button is clicked', function () {
```

Comment:
Yes, makes sense. I was following how the tests for sms and call were named. I'll change this.